src/ai-dialog-footer.ts

```diff
@@ -1,44 +1,46 @@
+import {inject} from 'aurelia-dependency-injection';
```

Comment:
I typically favor `static inject = []` over the decorator, not sure if there is any real speed implication but for less imports but that's more of a style choice just pointing it out.